crates/sui-core/src/authority_server.rs

```diff
@@ -840,6 +856,18 @@ impl ValidatorService {
         self.soft_bundle_validity_check(&certificates, &epoch_store)
             .await?;
 
+        info!(
```

Comment:
Usually Sui logs at debug level per request / transaction. But this can be fine because of the low initial TPS.